sysrib/server_test.go

```diff
@@ -156,6 +156,173 @@ func configureInterface(t *testing.T, intf *AddIntfAction, yclient *ygnmi.Client
 	}
 }
 
+var (
+	// v4v6Mapper maps IPv4 addresses to IPv6 addresses for testing v6.
+	v4v6Mapper = map[string]string{
```

Comment:
can this be func instead a hardcoded map?

Reply:
`mapAddress` uses this map to convert the addresses.

Reply:
Done.